README.rst

```diff
@@ -416,13 +416,19 @@ get_endpoint utility to do so:
 
     # '/buckets/payments/collections/receipts/records/c6894b2c-1856-11e6-9415-3c970ede22b0'
 
-Handling YAML date objects
---------------------------
+
+Handling datetime and date objects
+----------------------------------
 
 JSON doesn't have a specific type cast for dates while YAML does.
 
 In case a YAML file contains a date or datetime object, it will be
```

Comment:
There's still mentions of YAML here.

Reply:
Should I keep this readme paragraph at all then?

Reply:
I wasn't sure what paragraph you meant, so I just made some more comments about the paragraph that still exists.